package.json

```diff
@@ -53,26 +53,28 @@
     "dist/"
   ],
   "dependencies": {
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "http": "0.0.1-security",
-    "https": "^1.0.0",
+    "@fortawesome/fontawesome-svg-core": "^1.2.35",
+    "@fortawesome/free-solid-svg-icons": "^5.15.3",
+    "@fortawesome/react-fontawesome": "^0.1.14",
     "axios": "^0.21.1",
     "body-parser": "^1.19.0",
+    "http": "0.0.1-security",
+    "https": "^1.0.0",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
     "stream": "0.0.2",
     "url-loader": "^4.1.1",
     "xmldom": "^0.6.0",
-    "xpath": "0.0.32",
-    "@fortawesome/fontawesome-svg-core": "^1.2.35",
-    "@fortawesome/free-solid-svg-icons": "^5.15.3",
-    "@fortawesome/react-fontawesome": "^0.1.14"
+    "xpath": "0.0.32"
   },
   "devDependencies": {
     "@babel/preset-react": "^7.0.0",
     "@osjs/dev-meta": "^1.0.3",
     "babel-eslint": "^10.1.0",
     "eslint": "^7.30.0",
-    "eslint-plugin-react": "^7.24.0"
+    "eslint-plugin-react": "^7.24.0",
+    "webpack": "^4.46.0",
```

Comment:
We normally don't add `webpack` and `webpack-cli` as a `devDependency` of packages. They are already added in [@osjs/dev-meta](https://github.com/os-js/osjs-dev-meta/blob/b290b63976d72936bf5c6fcd241d641b06d3091b/package.json#L46) package

Reply:
It should be added. Because this project needs `webpack` itself and it should not depend on `webpack` installation based on another project.

Reply:
As you can see [here](https://github.com/os-js/osjs-preview-application/blob/de9e58f7e0f3b5f81f4ce6d30e68fefe4c0eb642/package.json#L37) and [here](https://github.com/os-js/osjs-filemanager-application/blob/5ad4859d073b4be374ad98d1fe3a110166325f73/package.json#L37), these sample applications also do the same. So I think we'd better to follow the overall behaviour.